hs_core/views/__init__.py

```diff
@@ -480,10 +480,16 @@ def my_resources(request, page):
     user = request.user
     # get a list of resources with effective OWNER privilege
     owned_resources = user.uaccess.get_resources_with_explicit_access(PrivilegeCodes.OWNER)
+    # remove obsoleted resources from the owned_resources
+    owned_resources = owned_resources.exclude(object_id__in=Relation.objects.filter(type='isReplacedBy').values('object_id'))
```

Comment:
@pkdash I thought about doing this 'exclude" within the method ```get_resources_with_explicit_access()``` initially, but then I realized it is confusing in logic if I exclude obsoleted resources inside the method since the user access control privileges are not changed for obsoleted resources and thus this method should return obsoleted resources for resource list with specified privilege. In fact, the versioning and obsolecense chain is maintained through Relation metadata elements which does not change access control privileges in any way. Since we decided to exclude obsoleted resources from My Resources page, I exclude these obsoleted resources just for My Resources listing. As Dave suggested, in the future, we may add a "Show all versions" functionality on My Resources page, so we should not exclude those obsoleted resources inside the method, but instead doing what we need outside the method depending on specific requirements, which explains why I did it this way. Let me know what you think.